### ISSUE TITLE
LMarker and LIcon render very slowly using Vue 3 with Vite

### ISSUE THREAD

**Issue by 4000degrees** [vue-leaflet/vue-leaflet#204]
Markers render slowly and consecutively as seen in the gif. This behavior is observed only in a vite project, and not in webpack. I have prepared two demos with the same code but different setups: [vite](https://github.com/4000degrees/vue-leaflet-marker-vite-demo) and [webpack](https://github.com/4000degrees/vue-leaflet-marker-webpack-demo). Sorry, couldn't get a codesandbox running with the vite demo.

The markers render with default icons, but then they are replaced with custom HTML icons. I think this problem is the same one as described in issues [170](https://github.com/vue-leaflet/vue-leaflet/issues/170), [172](https://github.com/vue-leaflet/vue-leaflet/pull/172) and [173](https://github.com/vue-leaflet/vue-leaflet/pull/173) but here its even more pronounced.

![vue-leaflet-marker-vite-demo](https://user-images.githubusercontent.com/42625050/161158564-d2327edd-9447-4e3d-9f07-d4402a280d42.GIF)

**Comment by isuke01** [comment]
Yep, also this is why I switched to native leaflet after some extensive tests :/

**Comment by msslgomez** [comment]
@isuke01 Could you post an example of switching to native leaflet?
